src/jsdoc.ts

```diff
@@ -270,7 +264,12 @@ export function toString(tags: Tag[], escapeExtraTags: string[] = []): string {
 
   let out = '';
   out += '/**\n';
+  const emitted = new Set<string>();
   for (let tag of tags) {
+    if (emitted.has(tag.tagName) && SINGLETON_TAGS.has(tag.tagName)) {
```

Comment:
When trying to fix this, my impression was that overall our approach to JSDoc is too simplistic. We should have a proper model class with an appropriate data structure backing it. I've added this comment on parse above, which seems the closest place:

```javascript
// TODO(martinprobst): representing JSDoc as a list of tags is too simplistic. We need functionality
// such as merging (below), de-duplicating certain tags (@deprecated), and special treatment for
// others (e.g. @suppress). We should introduce a proper model class with a more suitable data
// strucure (e.g. a Map<TagName, Values[]>).
```